packages/react-scripts/package.json

```diff
@@ -82,8 +74,8 @@
     "workbox-webpack-plugin": "4.3.1"
   },
   "devDependencies": {
-    "react": "^16.11.0",
-    "react-dom": "^16.11.0"
+    "react": "16.4.2",
```

Comment:
Worth aligning to the version of React currently used in production: 16.2.0

Reply:
This was taken from BRS 7 so that means that version was wrong too

Reply:
Locked the version to correct prod version